### DISCUSSION TITLE
api rule

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#3630] [Q&A]
suppose i have two collection 
one session second transaction
in transaction collection if want to set a create request like @request.data.session = session collection key field if both match than can create

**Comment by ganigeorgiev** [comment]
If it is not directly related you can use the `@collection.*` rule, aka. `@request.data.session ?= @collection.sessions.id`.
It roughly translates to _"require the submitted session field value to match with any/at-least-one record id from the sessions collection"._
